packages/video-intelligence/src/index.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ 'use strict';
+
+ var gapic = {
+   v1beta1: require('./v1beta1'),
+ };
+
+ const VERSION = require('../package.json').version;
+
+
+ /**
+  * Create an imageAnnotatorClient with additional helpers for common
```

Comment:
It doesn't look like we have additional helpers, so I don't think we would want to include this in the docs.